test-data/unit/check-generic-alias.test

```diff
@@ -44,7 +44,7 @@ t10: collections.ChainMap[int, str]  # E: "ChainMap" is not subscriptable, use "
 
 
 [case testGenericBuiltinFutureAnnotations]
-# flags: --python-version 3.7
+# flags: --python-version 3.8
```

Comment:
I think the flags can also be removed here since this should work the same on all supported versions

Reply:
```suggestion
```